Sources/Widgets/Widgets3D/ResliceCursorWidget/Constants.js

```diff
@@ -4,6 +4,14 @@ export const ScrollingMethods = {
   RIGHT_MOUSE_BUTTON: 2,
 };
 
+// Note: These strings are exactly method's name in ResliceCursorWidget/behavior.js
```

Comment:
Additionally, in behavior.js you could do
`publicAPI.translateCenter = `-> `publicAPI[InteractionMethodsName.TranslateAxis] = `